README.md

```diff
@@ -25,10 +26,41 @@ $ export REGISTRY=<SOME_REGISTRY> \
 ## Using the Marketplace Operator
 
 ### Description
+The operator manages two CRDs: [OperatorSource](./deploy/crds/operatorsource.crd.yaml) and [CatalogSourceConfig](./deploy/crds/catalogsourceconfig.crd.yaml).
 
-The marketplace operator manages two CRDs: [OperatorSource](./deploy/crd/operatorsource.crd.yaml) and [CatalogSourceConfig](./deploy/crd/catalogsourceconfig.crd.yaml). When an OperatorSource CR is created in the same namespace as where the marketplace operator is running (we recommend the namespace be called "openshift-marketplace"), the operator will download artifacts stored in the registry specified in this OperatorSource CR (for now, please see documentation about using [quay](https://quay.io)'s appregistry API). For an example of this OperatorSource CR please see the [examples](./deploy/examples/) folder.
+`OperatorSource` is used to define the external datastore we are using to store operator artifacts. At the moment we only support Quay's app-registry as our external datastore. Here is an example `OperatorSource`:
+```
+apiVersion: marketplace.redhat.com/v1alpha1
+kind: OperatorSource
+metadata:
+  name: my-operators
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: my-quay-appregistry-namesapce
+  displayName: "My Operators"
+  publisher: "Your name or company"
```

Comment:
Does it make sense to put this yaml in the readme? I might recommend having a link to our deploy/examples/ folder instead? This might be hard to maintain as another point of failure if/when we modify the crd. Or, at the very least, have this in a separate doc?

Reply:
I added this as an explanation for what an `OperatorSource` looks like and what the `spec` values should be. And yes, we need to maintain documentation when we change things.

Reply:
I have removed the yaml and added explanation for the spec fields.